### DISCUSSION TITLE
Get data via "Datenzugriffe"

### DISCUSSION THREAD

**Discussion by bop17000** [DarwinsBuddy/WienerNetzeSmartmeter#145] [Ideas]
Hi!

I recently joined an app called nista.io.
There, you put in your Zählerpunkt-ID and after some time, you will get an request in the smartmeter web portal. If you accept the request your data will be transferred to the service.
Maybe this can be utilized here too?

Greetings

**Comment by DarwinsBuddy** [comment]
Thank you for your input. I guess they've gotten a direct B2B integration somehow, which would only be doable with some infrastructure and a lot more time and effort provided. Since this is a community-driven hobby project, I fear we can not provide the service this way.

Thank you for your input though. Very much appreciated. I got to know nista.io :)
